cmd/clone/clone.go

```diff
@@ -113,6 +113,27 @@ func run(c *cobra.Command, _ []string) {
 			continue
 		}
 		createBranchActivity.EndWithSuccess()
+
+		if nofork {
```

Comment:
i got confused with this for a minute.
Wouldn't it be clearer if you put the block below in a `if !nofork {` statement?

Reply:
Yeah can do, I somehow thought it was neater to reduce the amount of logic within the conditional block. But see your point - will do it that way instead.